zebra-test/Cargo.toml

```diff
@@ -31,3 +31,4 @@ tracing = "0.1.31"
 
 [dev-dependencies]
 tempfile = "3.3.0"
+
```

Comment:
Minor: Can you revert this change before merging, it might cause merge conflicts?

As a longer-term fix, can you open a ticket to add the relevant workflow and docker files to the workflow path filters?